test/built-ins/TypedArrays/of/invoked-as-method.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+id: sec-%typedarray%.of
+description: >
+  "of" cannot be invoked as a method of %TypedArray%
+info: >
+  22.2.2.2 %TypedArray%.of ( ...items )
+
+  ...
+  5. Let newObj be ? TypedArrayCreate(C, «len»).
+  ...
+
+  22.2.4.6 TypedArrayCreate ( constructor, argumentList )
+
+  1. Let newTypedArray be ? Construct(constructor, argumentList).
+  ...
+includes: [testTypedArray.js]
+---*/
+
+testWithTypedArrayConstructors(function(TA) {
+  assert.throws(TypeError, function() {
+    TA.of();
+  });
+});
```

Comment:
Can you explicitly cite the spec that this is testing—I don't think this should throw any errors, right? It should just produce an empty %TypedArray%